hw3/01-characters.js

```diff
@@ -2,3 +2,55 @@
 const url = 'https://thronesapi.com/api/v2/Characters';
 
 fetch(url);
+
+const characterFlex = document.getElementById("charcter-items");
+fetch(url)
+  .then((response) => response.json())
+  .then((data) => {
+    data.forEach((c) => {
```

Comment:
"c" is not a meaningful name. Consider changing it to something more descriptive.